test/clj_http/test/conn_mgr_test.clj

```diff
@@ -181,7 +181,8 @@
   (let [regular (conn-mgr/make-regular-conn-manager {})
         regular-reusable (conn-mgr/make-reusable-conn-manager {})
         async (conn-mgr/make-regular-async-conn-manager {})
-        async-reusable (conn-mgr/make-reuseable-async-conn-manager {})]
+        async-reusable (conn-mgr/make-reusable-async-conn-manager {})
+        async-reuseable (conn-mgr/make-reuseable-async-conn-manager {})]
```

Comment:
There should either be a second assertion below, or only the correctly spelled one should be tested here.